pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -734,7 +734,7 @@ ${e.message}
     );
     if (!success) {
       await dependenciesHashFile.delete();
-      return (success, kernelFile, dependenciesHashFile);
+      return (success, kernelFile, dependenciesHashes);
```

Comment:
It is rather strange that if we failed to compile the hook, we still return a `Uri` to a kernel file (which hopefully doesn't exist) and a dependencies hashes file (which also hopefully doesn't exist).

Wouldn't it be cleaner to return `null` signaling to the caller that the hook wasn't compiled and it's not safe to use kernel file or hashes file?